web/web.go

```diff
@@ -172,42 +171,43 @@ func Run(host string, routes http.Handler, readTimeout, writeTimeout time.Durati
 		MaxHeaderBytes: 1 << 20,
 	}
 
-	// We want to report the listener is closed.
-	var wg sync.WaitGroup
-	wg.Add(1)
+	// We want to use an error channel to block and receive the error.
+	serverErrors := make(chan error, 1)
+	defer close(serverErrors)
```

Comment:
Technically, we don't need to close the channel. Is it a good practice to `defer close`?

Reply:
Don't close the channel unless you need to signal without data. So remove that code if it is not necessary.